File: src/main/java/net/dontdrinkandroot/gitki/wicket/event/FileDeletedEvent.java
package net.dontdrinkandroot.gitki.wicket.event;

import net.dontdrinkandroot.gitki.model.FilePath;
import org.apache.wicket.ajax.AjaxRequestTarget;

/**
 * @author Philip Washington Sorst <philip@sorst.net>
 */
public class FileDeletedEvent extends FileEvent
{
    public FileDeletedEvent(FilePath filePath)
    {
        super(filePath);
    }

    public FileDeletedEvent(FilePath filePath, AjaxRequestTarget target)
    {
        super(filePath, target);
    }
}

File: src/main/kotlin/net/dontdrinkandroot/gitki/wicket/event/FileDeletedEvent.kt
package net.dontdrinkandroot.gitki.wicket.event

import net.dontdrinkandroot.gitki.model.FilePath
import org.apache.wicket.ajax.AjaxRequestTarget

class FileDeletedEvent : FileEvent {
    constructor(filePath: FilePath) : super(filePath)
    constructor(filePath: FilePath, target: AjaxRequestTarget?) : super(filePath, target)
}